src/cart.scss

```diff
@@ -83,8 +99,67 @@ body {
     border-radius: 100%;
     border: 0.2rem solid #49474d;
   }
+
+  .subtotal-count {
+    display: flex;
+    gap: 1.5rem;
+    margin: var(--cart-item-margin);
+    margin-left: 0.3rem;
+  }
+
+  .update-cart-btn {
+    display: flex;
+    flex: 1;
+    justify-content: flex-end;
+
+    .update-btn {
+      margin: 2% 5% 2% 0%;
+      width: 8rem;
+      border-radius: 16%;
+      border: 0.2rem solid #49474d;
+    }
+  }
+
+  hr#shopping-cart-line-end {
+    border: none;
+    height: 0.5rem;
+    margin: 1rem 0rem 1rem 12rem;
+    background: linear-gradient(to left, #333, transparent);
+  }
 }
 
-:root {
-  --cart-item-margin: 2rem 0rem 2rem 0rem;
+#summary-container {
+  display: flex;
+  flex-direction: column;
+  border: 0.2rem solid var(--border-color);
+  border-radius: var(--container-border-radius);
+  max-width: var(--container-max-width);
+  margin: auto;
+  margin-top: var(--margin-top);
+  padding: var(--container-padding);
+
+  h1 {
+    @include flex-center;
+  }
+
+  #summary-title-line {
+    border: none;
+    height: var(--line-height);
+    margin: 0rem 0rem 1rem 0rem;
+    background: linear-gradient(to bottom, var(--line-color), transparent);
+  }
+
+  .summary-break-line {
+    border: none;
```

Comment:
I see many different <hr> elements, each with a different class but they have things in common. 

So the classes should be broken appart. However, I see that all classes include in css, border: none. So the question is, why would you introduce an element such as <hr> which, by default is kind of like a border and then remove it from css?